### PR TITLE
Refactor CoinHandler and MeetingHandler files

### PR DESCRIPTION
**Pull request by Ajkunas**
Brief refactoring of `handleCreateMeeting` and `handlePostTransaction` functions.

### CODE REVIEW

File: be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/MeetingHandler.scala [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
 
   def handleCreateMeeting(rpcMessage: JsonRpcRequest): GraphMessage = {
-    rpcMessage.getParamsChannel.decodeChannelLaoId match {
-      case Some(_) =>
-        val ask: Future[GraphMessage] = dbAskWritePropagate(rpcMessage)
-        Await.result(ask, duration)
-      case _ => Right(PipelineError(
-          ErrorCodes.INVALID_DATA.id,
-          s"Unable to create meeting: invalid encoded laoId '${rpcMessage.getParamsChannel}'",
-          rpcMessage.id
```
Comment (pierluca) [inline be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/MeetingHandler.scala:18]: here as well, having this split on multiple lines was helpful to read it quickly.

**Review by pierluca — APPROVED** [review]
Looks good to me, just be mindful that removing too many newlines might hurt readability a bit.
In general it's preferable to keep line length below some limit, usually 80 to 120, depending on the organization you work in 😄 
According to the Scala editor config, this should be 120: 
https://github.com/dedis/popstellar/blob/master/be2-scala/.editorconfig#L7

File: be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/MeetingHandler.scala (hunk 1 above)
Comment (Ajkunas) [reply be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/MeetingHandler.scala:18]: Understood, I agree that it is easily readable.

**Review by pierluca — APPROVED** [review]
Good enough for now 😁

**Pull request merged**